src/expression/node/Node.js

```diff
@@ -369,11 +370,9 @@ export const createNode = /* #__PURE__ */ factory(name, dependencies, ({ mathWit
    * @param {Object} scope
    */
   function _validateScope (scope) {
-    for (const symbol in scope) {
-      if (hasOwnProperty(scope, symbol)) {
-        if (symbol in keywords) {
-          throw new Error('Scope contains an illegal symbol, "' + symbol + '" is a reserved keyword')
-        }
+    for (const symbol of getSafeProperties(scope)) {
+      if (symbol in keywords) {
+        throw new Error('Scope contains an illegal symbol, "' + symbol + '" is a reserved keyword')
```

Comment:
This checks for against all properties in `keywords`, including on the prototype.

I'm nervous about changing this too soon, although *I think* if the `scope` is map-like, we could allow the more risqué names.

Reply:
This is helpful context, thanks.